responsibleai/tests/rai_insights/test_rai_insights_validations.py

```diff
@@ -377,6 +378,52 @@ def test_classes_exceptions(self):
         assert 'The train labels and distinct values in ' + \
             'target (train data) do not match' in str(ucve.value)
 
+        X_train[TARGET] = y_train
+        X_test[TARGET] = y_test
+
+        with pytest.raises(UserConfigValidationException) as ucve:
+            RAIInsights(
+                model=model,
+                train=X_train,
+                test=X_test,
+                target_column=TARGET,
+                task_type='classification',
+                classes=[0, 1],
+                feature_metadata=FeatureMetadata(dropped_features=[
+                    "mean radius",
+                    "mean texture",
+                    "mean perimeter",
+                    "mean area",
+                    "mean smoothness",
+                    "mean compactness",
+                    "mean concavity",
+                    "mean concave points",
+                    "mean symmetry",
+                    "mean fractal dimension",
+                    "radius error",
+                    "texture error",
+                    "perimeter error",
+                    "area error",
+                    "smoothness error",
+                    "compactness error",
+                    "concavity error",
+                    "concave points error",
+                    "symmetry error",
+                    "fractal dimension error",
+                    "worst radius",
+                    "worst texture",
+                    "worst perimeter",
+                    "worst area",
+                    "worst smoothness",
+                    "worst compactness",
+                    "worst concavity",
+                    "worst concave points",
+                    "worst symmetry",
+                    "worst fractal dimension",
```

Comment:
Could you also add a test when x_train and x_test are empty that is they don't have any features?